support/docker/dev/Dockerfile

```diff
@@ -15,10 +15,22 @@ WORKDIR /home/user/PeerTube
 # Configure Cloud9 IDE to use PeerTube's source directory as workspace (-w).
 RUN sudo sed -i "s/-w \/home\/user/-w \/home\/user\/PeerTube/" /etc/supervisord.conf
 
+# Install dependencies.
+RUN yarn install
+
 # Configure Janitor for PeerTube.
 ADD janitor.json /home/user/
 RUN sudo chown user:user /home/user/janitor.json
 
 # Configure and build PeerTube.
-RUN yarn install \
- && npm run build
+ADD create_user.sql /tmp/
+RUN sudo service postgresql start && \
+    sudo -u postgres psql --file=/tmp/create_user.sql && \
+    sudo rm /tmp/create_user.sql
```

Comment:
Nit: Even if you delete files here, they'll remain part of the image history, since they were `ADD`ed above. I suggest leaving this file somewhere useful, or running these steps with inline psql commands as opposed to a file.

Also, if this is really necessary to get PeerTube to work, why isn't it in a `Makefile` or in the `package.json` NPM scripts?

Reply:
Re: files: well, these are tiny, so I don't seem to care too much ;) I'll just let them in /tmp in this case.

Re: Makefile/package.json docs: it depends a lot on your database install / version; it's more of an external dependency that something mandated by PeerTube, so IMO it makes sense to keep it separate.